packages/react-native-bpk-component-horizontal-nav/src/BpkHorizontalNav.js

```diff
@@ -104,7 +104,8 @@ class BpkHorizontalNav extends React.Component<
     };
   }
 
-  componentWillReceiveProps(nextProps) {
+  // eslint-disable-next-line camelcase
+  UNSAFE_componentWillReceiveProps(nextProps) {
```

Comment:
Should we fix this now to use the correct methods?

Reply:
We should fix this as soon as we can, but I don't think we should as part of this update because is more changes to introduce errors in an already big change. So I'd rather do the RN update first and then look at removing these methods.

Reply:
Ok sounds good - we should make an issue for this 😄

Reply:
We have one in the backlog already to remove deprecated methods :)